### DISCUSSION TITLE
saving

### DISCUSSION THREAD

**Discussion by ianfreya8** [brackets-cont/brackets#143] [Q&A]
hello, I am a new user on brackets.io and i was wondering if there was a way to make a custom project as whenever I tried to save edit and change a premade on (ice a .js file from my Udemy course) it says that that I do not have permission to edit and save this project. if anyone has ideas then please help me.

**Comment by charlypa** [comment]
@ianfreya8 
Try creating a folder in your home directory and save your files there

**Reply by ianfreya8** [reply]
i tried just now but after that when i made a project in it with my own simple code it said that i needed an "index.html" in my project

**Reply by charlypa** [reply]
@ianfreya8
if you have an Html file in that folder. Please rename it to index.html
